src/plattenalbum.py

```diff
@@ -1038,8 +1038,8 @@ class SetupDialog(ConnectDialog):
 	def __init__(self):
 		super().__init__(_("Setup"), GLib.Variant("b", False))
 		box=Gtk.Box(orientation=Gtk.Orientation.VERTICAL, spacing=12)
-		box.append(Gtk.Label(label=_("After installing the “Music Player Daemon” (<tt>mpd</tt>) the following two commands"\
```

Comment:
The "two" here conflicts with master since I forgot a third command. So this needs to be resolved.

Reply:
If you don't have time, I can do this as well.

Reply:
I just dropped the count altogether.

Reply:
That's also a good solution. But the conflict still remains :-) I'll fix it.